dev-libs/zziplib/zziplib-0.13.71-r3.ebuild

```diff
@@ -0,0 +1,73 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7,8,9} )
+inherit cmake flag-o-matic python-any-r1
+
+DESCRIPTION="Lightweight library for extracting data from files archived in a single zip file"
+HOMEPAGE="http://zziplib.sourceforge.net/"
+SRC_URI="https://github.com/gdraheim/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="|| ( LGPL-2.1 MPL-1.1 )"
+SLOT="0/13"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~mips ~ppc ~ppc64 ~s390 ~sparc ~x86 ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos ~x86-macos ~sparc-solaris ~sparc64-solaris ~x64-solaris ~x86-solaris"
+IUSE="doc sdl static-libs test"
+
+RESTRICT="!test? ( test )"
+
+BDEPEND="
+	doc? (
+		${PYTHON_DEPS}
+	)
+	test? (
+		${PYTHON_DEPS}
+		app-arch/zip
+	)
+"
+DEPEND="
+	sys-libs/zlib
+	sdl? ( >=media-libs/libsdl-1.2.6 )
+"
+RDEPEND="${DEPEND}"
+
+PATCHES=(
+	"${FILESDIR}"/${PN}-0.13.69-009-perror.patch
+	"${FILESDIR}"/${PN}-0.13.71-testbuilds-opensuse15-ninja-sdl2.patch
+	"${FILESDIR}"/${PN}-0.13.71-shell-DESTDIR.patch
+	"${FILESDIR}"/${PN}-0.13.71-make_am-make_cm.patch
+	"${FILESDIR}"/${PN}-0.13.71-reorganize-ZZIPOPTIONFLAGS.patch
+	"${FILESDIR}"/${PN}-0.13.71-manvolnum.patch
+	"${FILESDIR}"/${PN}-0.13.71-add-ZZIP_INSTALL_BINS-cmake-option.patch
+	"${FILESDIR}"/${PN}-0.13.71-open-file-read.patch
+	"${FILESDIR}"/${PN}-0.13.71-fix-compile-with-ZZIPFSEEKO-OFF.patch
+	"${FILESDIR}"/${PN}-0.13.71-join-paths-pc-zzipsdldir.patch	# Merge pull request 89
+	"${FILESDIR}"/${PN}-0.13.71-add-downloaddir-and-downloadonly.patch
+	"${FILESDIR}"/${PN}-0.13.71-allow-to-run-zziptests.py.patch
+	"${FILESDIR}"/${PN}-0.13.71-ExternalProjects_add-zziptestsdownload.patch
+	"${FILESDIR}"/${PN}-0.13.71-skip-CVE-tests-if-file-can-not-be-downloaded.patch
+	"${FILESDIR}"/${PN}-0.13.71-allow-for-cmake-DZZIP_TESTCVE-OFF.patch
+	"${FILESDIR}"/${PN}-0.13.71-check-for-cmake-DZZIP_TESTCVE-OFF.patch
+	"${FILESDIR}"/${PN}-0.13.71-allow-to-pass-downloads-ZZIP_TESTCVE.patch
+	"${FILESDIR}"/${PN}-0.13.71-check-to-pass-downloads-ZZIP_TESTCVE.patch
+	"${FILESDIR}"/${PN}-0.13.71-run-azure-pipelines-with-DZZIP_TESTCVE-OFF.patch
+	"${FILESDIR}"/${PN}-0.13.71-copy-directory-instead-of-unpacking-manpages.patch
+	"${FILESDIR}"/${PN}-0.13.71-test-with-cm-install.patch
+	"${FILESDIR}"/${PN}-0.13.71-installing-man3-pages.patch
```

Comment:
So many patches... Are they all upstream? Can you convince upstream to make a new release?

Reply:
As stated in the commit message, they (except only two of them) are from upstream's develop branch https://github.com/gdraheim/zziplib/compare/v0.13.71...develop
No upstream activity since April 19, 2020